checkST2110.js

```diff
@@ -28,8 +28,8 @@ const groupPattern = /a=group:DUP\s+(\S+)\s+(\S+)/;
 const ssrcPattern = /a=ssrc:(\d+)\s/;
 const videoPattern = /video\s+(\d+)(\/\d+)?\s+(RTP\/S?AVP)\s+(\d+)/;
 const rtpmapPattern = /a=rtpmap:(\d+)\s(\S+)\/(\d+)\s*/;
-const fmtpPattern = /a=fmtp:(\d+)\s+(?:([^\s=;]+)(?:=([^\s;]+))?;\s+)*$/;
-const fmtpParams = /([^\s=;]+(?:=[^\s;]+)?);/g;
+const fmtpPattern = /a=fmtp:(\d+)\s+(?:([^\s=;]+)(?:=([^\s;]+))*(?:;\s*)*)+$/;
```

Comment:
Is this right? The `(?:;\s*)*` matches zero or many semi-colons... Oh, I guess the greedy matching of the previous terms means you can't actually have zero semi-colons anywhere but right at the end, and multiple semi-colon separators are perhaps allowed... OK...

Reply:
I don't see a problem with changing that to `(?:;\s*){0,1}`. Certainly a quick test shows that avoids `a=fmtp:79 foo=bar;;;;foo=bar` from passing.